definitions/install.yaml

```diff
@@ -386,6 +386,8 @@ spec:
           properties:
             policy:
               type: string
+            synchronize:
+              type: boolean
```

Comment:
It needs to be added to all CRD definitions, kustomize base CRD, Helm Charts.

Reply:
yes